crates/frontend/src/components/input_components.rs

```diff
@@ -0,0 +1,86 @@
+use leptos::*;
+use serde_json::{Map, Value};
+
+use crate::components::dropdown::{Dropdown, DropdownBtnType, DropdownDirection};
+
+#[component]
+pub fn enum_dropdown(
+    schema: Map<String, Value>,
+    config_value: String,
+    handle_change: Callback<String, ()>,
+    #[prop(default = String::new())] class: String,
+    #[prop(default = false)] disabled: bool,
+) -> impl IntoView {
+    let (value, set_value) = create_signal(config_value.replace("\\", ""));
```

Comment:
Can we avoid replacing it ? 
What if someone actually wants to send this in the input ?